warehouse/integrations/datalake/datalake_test.go

```diff
@@ -408,3 +344,54 @@ func TestIntegration(t *testing.T) {
 		testhelper.VerifyConfigurationTest(t, dest)
 	})
 }
+
+func createGCSBucket(t testing.TB, ctx context.Context, endpoint, bucketName string) {
+	t.Helper()
+
+	require.NoError(t, testhelper.WithConstantRetries(func() error {
```

Comment:
Can we use `require.Eventually` instead that is easier to configure and has an algorithm that would allow the test to potentially finish sooner?

Reply:
Same in the other occurrence below.